src/library/Uno.Material/Styles/Controls/v2/ToggleButton.xaml

```diff
@@ -372,15 +372,15 @@
 										<Setter Target="ContentPresenter.BorderBrush" Value="{ThemeResource TextToggleButtonBorderBrushChecked}" />
 										<Setter Target="Root.Background" Value="{ThemeResource TextToggleButtonBackgroundChecked}" />
 										<Setter Target="Root.BorderBrush" Value="{ThemeResource TextToggleButtonBorderBrushChecked}" />
-										<Setter Target="AlternateContentPresenter.Opacity" Value="{Binding Path=(um:ControlExtensions.AlternateContent), RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource NullToTransparentConverter}, FallbackValue=1, TargetNullValue=1}" />
+										<Setter Target="AlternateContentPresenter.Opacity" Value="{Binding Path=(ut:ControlExtensions.AlternateContent), RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource NullToTransparentConverter}, FallbackValue=1, TargetNullValue=1}" />
```

Comment:
undeclared `ut` xmlns in this file
this type of error wont be caught be the compiler i think
until runtime 💀

Reply:
invalidate in other files for same problem too

Reply:
Validated